components/system/components/GlobalModal.js

```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+import * as Constants from "~/common/constants";
+import * as SVG from "~/components/system/svg";
+import * as Strings from "~/common/strings";
+
+import { css } from "@emotion/react";
+
+const STYLES_BACKGROUND = css`
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100vw;
+  height: 100vh;
```

Comment:
✂️  remove this `height: 100vh`, go with `display: flex; align-items: center; justify-content: center;` and all children will be centered, you won't need to have a modal thats absolute positioned as the child.